package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "catbox",
     "description": "Multi-strategy object caching service",
-    "version": "7.1.0",
+    "version": "7.1.1",
```

Comment:
Never change version numbers. It's up to the maintainer to figure that out when publishing. This just causes pain when merging.